src/Builder.php

```diff
@@ -188,6 +188,20 @@ public function withGlobalScope($identifier, array $scope)
         return $this;
     }
 
+    /**
+     * Remove a registered global scope.
+     *
+     * @param  string  $scope
+     * @return $this
+     */
+    public function withoutGlobalScope($scope)
+    {
+        unset($this->scopes[$scope]);
+        $this->removedScopes[] = $scope;
```

Comment:
Are you sure that works ? withGlobalScope currently add scope as query on line 186

Reply:
Updated method withGlobalScope and getQuery to apply global scopes just before calling the API.